pkg/users/local.js

```diff
@@ -102,7 +103,14 @@
             <AccountDetails accounts={accountsInfo} groups={groupsExtraInfo} shadow={shadow || []}
                             current_user={current_user_info && current_user_info.name} user={path[0]} />
         );
-    }
+    } else if (path.length === 2 && path[0] === "group" && groupsExtraInfo.length > 0 && accountsInfo.length > 0) {
+        const current_group = groupsExtraInfo.find(g => g.name == path[1]);
+        if (current_group) {
+            return (
+                <GroupDetails accounts={accountsInfo} groups={groupsExtraInfo} group={path[1]} />
+            );
+        }
+    } else return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230213-073621-682da179-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)